driver/pace/driver/gridconfig.py

```diff
@@ -1,23 +1,23 @@
 import dataclasses
 from typing import Optional
 
-import numpy as np
-
 
 @dataclasses.dataclass
 class GridConfig:
-    stretch_mode: bool = False
-    stretch_factor: Optional[np.float_] = None
-    lon_target: Optional[np.float_] = None
-    lat_target: Optional[np.float_] = None
+    stretch_grid: bool = False
```

Comment:
If we're adding a class to configure the grid initialization, it should contain the code to initialize the grid. We should move the code out of the state-initialization classes into this. There would be two implementations - serialized grid, and generated grid.